### PR TITLE
Cbc syslog ea1234

### PR DESCRIPTION
**Pull request by kgarcia-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [X] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [X] Code follows the style guidelines of this project (PEP8, clean code).
- [X] Linter has passed locally and any fixes were made for failures.
- [X] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

- Ticket Number: EA-16134

## Pull Request Description
Replaced a built in package that was only compatible with Linux environments. replaced it with a package that was compatible with all platforms.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

If this introduces a breaking change, please describe the impact and migration path for existing applications below.

## How Has This Been Tested?

Tests were ran locally in a Windows, Linux and MacOS environment, End to End testing was also completed

### CODE REVIEW

File: PIP_README.md [hunk 1]
```diff
@@ -234,5 +232,17 @@ specifications. Below is a table of all the configurable inputs that can be used
     #[cbdefenseserver2]
     #api_connector_id = KJARWBZ111
     #api_key = CQF35EIH2WDF69PTWKGC4111
+    #siem_connector_id = UEUWR4U111
+    #siem_api_key = XNS5UKWZXZMCC3CYC7DFM111
```
Comment (dly-cb) [inline PIP_README.md:236]: Are these our real API/SIEM keys?? Should we consider utilizing a fake one?

File: README.md [hunk 2]
```diff
@@ -241,5 +241,12 @@ specifications. Below is a table of all the configurable inputs that can be used
     #[cbdefenseserver2]
     #api_connector_id = KJARWBZ111
     #api_key = CQF35EIH2WDF69PTWKGC4111
+    #siem_connector_id = UEUWR4U111
+    #siem_api_key = XNS5UKWZXZMCC3CYC7DFM111
```
Comment (dly-cb) [inline README.md:245]: Same here, for question above

File: PIP_README.md (hunk 1 above)
Comment (kgarcia-cb) [reply PIP_README.md:236]: No they are fake.

File: README.md (hunk 2 above)
Comment (kgarcia-cb) [reply README.md:245]: Yeah they are all Fake Keys!

Comment (dly-cb) [reply README.md:245]: Awesome thanks for the check :)

File: PIP_README.md (hunk 1 above)
Comment (dly-cb) [reply PIP_README.md:236]: Awesome, thanks for the check :)

**Pull request merged**